core/filter_api.php

```diff
@@ -3389,7 +3389,13 @@ function filter_draw_selection_area2( $p_page_number, $p_for_screen = true, $p_e
 
 		<!-- Match Type -->
 		<tr class="row-1">
-			<td class="small-caption category2"><a href="<?php echo $t_filters_url . FILTER_PROPERTY_MATCH_TYPE;?>" id="match_type_filter"><?php echo lang_get( 'filter_match_type' )?>:</a></td>
+			<td class="small-caption category2">
+				<a id="match_type_filter"
+					href="<?php echo $t_filters_url . FILTER_PROPERTY_MATCH_TYPE;?>"
+					<?php echo $t_dynamic_filter_expander_class; ?>>
+					<?php echo lang_get( 'filter_match_type_label' )?>
```

Comment:
filter_match_type lang string seems to be no longer being used.  Should we remove it?  Or do we expect to use it elsewhere?